pkg/daemon/ceph/client/mirror.go

```diff
@@ -157,7 +174,7 @@ func GetPoolMirroringStatus(context *clusterd.Context, clusterInfo *ClusterInfo,
 	logger.Debugf("retrieving mirroring pool %q status", poolName)
 
 	// Build command
-	args := []string{"mirror", "pool", "status", poolName}
+	args := []string{"mirror", "pool", "status", "--verbose", poolName}
```

Comment:
using `format=json` might be the best one here?

Reply:
We use it already `format=json`. 

 `--verbose` gives the mirrored images for the pool as well. 

``` sh-4.4$ rbd mirror pool status replicapool --verbose --format=json
{"summary":{"health":"OK","daemon_health":"OK","image_health":"OK","states":{"replaying":1}},"daemons":[{"service_id":"4339","instance_id":"4644","client_id":"a","hostname":"fv-az1195-222","ceph_version":"18.2.2","leader":true,"health":"OK"}],"images":[{"name":"test","global_id":"ebad309d-4d8f-4c6f-afe0-46e02ace26ac","state":"up+stopped","description":"local image is primary","daemon_service":{"service_id":"4339","instance_id":"4644","daemon_id":"a","hostname":"fv-az1195-222"},"last_update":"2024-03-18 04:16:54","peer_sites":[{"site_name":"4c3e1cb8-8129-43ab-8d75-abc20154fd4a","mirror_uuids":"a71727e9-63a4-4386-a44b-4cf48dc77fa8","state":"up+replaying","description":"replaying, {\"bytes_per_second\":0.0,\"bytes_per_snapshot\":0.0,\"last_snapshot_bytes\":0,\"last_snapshot_sync_seconds\":0,\"remote_snapshot_timestamp\":1710734899,\"replay_state\":\"idle\"}","last_update":"2024-03-18 04:16:36"}]}]}```